app/scripts/framework/service/meta-service.js

```diff
@@ -159,6 +159,9 @@ org.ekstep.services.metaService = new (org.ekstep.services.iService.extend({
      */
 	getFormConfigurations: function (data, callback) {
 		this.postFromService(this.getBaseURL() + this.getAPISlug() + this.getConfig('configEndPoint', '/data') + this.getConfig('configurationUrl', '/v1/form/read'), data, this.requestHeaders, callback)
-	}
+	},
 
+	getVideoLicense: function (data, callback) {
+		this.postFromService(this.getBaseURL() + this.getAPISlug() + '/asset/v3/license/validate ', data, this.requestHeaders, callback)
```

Comment:
Follow the same standard as we used for other plugins
this.getConfig('getVideoLicense', '/asset/v3/license/validate '),